src/backend/dx11/src/lib.rs

```diff
@@ -1697,6 +1702,8 @@ pub struct UnboundImage {
     usage: image::Usage,
     flags: image::StorageFlags,
     bind: d3d11::D3D11_BIND_FLAG,
+    compressed: bool,
```

Comment:
`format` already has this info, as I mentioned earlier, so no need to separately store it